projects/1.21/assets/super-factory-manager/sfm/lang/zh_cn.json

```
@@ -0,0 +1,238 @@
+  "gui.sfm.comfirm.funny.yes.3": "嗯，继续吧。",
+  "gui.sfm.comfirm.funny.yes.4": "尽管做。",
+  "gui.sfm.comfirm.funny.yes.5": "lol，干就完事了",
+  "gui.sfm.comfirm.funny.yes.6": "应用更改。",
+  "gui.sfm.container_inspector.container_slot_count": "容器槽位：%d",
+  "gui.sfm.container_inspector.inventory_slot_count": "物品栏槽位：%d",
+  "gui.sfm.container_inspector.mekanism_machine_inputs": "以下基于机器的输入配置",
+  "gui.sfm.container_inspector.mekanism_machine_outputs": "以下基于机器的输出配置",
+  "gui.sfm.container_inspector.mekanism_null_direction_warning": "通用机械模组方块在未设置方向时仅可读！！！！",
+  "gui.sfm.container_inspector.notice.1": "GUI槽位不一定总是对应自动化槽位！！！",
+  "gui.sfm.container_inspector.notice.2": "按 %s 以切换此覆盖层。",
+  "gui.sfm.container_inspector.show_exports_button": "导出检查器",
+  "gui.sfm.disk.tooltip.edit_in_hand": "手持磁盘时右击可直接编辑，无需管理器。",
+  "gui.sfm.exit_without_saving_confirm.message": "确定要放弃当前更改吗？",
+  "gui.sfm.exit_without_saving_confirm.no_button": "继续编辑",
+  "gui.sfm.exit_without_saving_confirm.title": "不保存直接退出？",
+  "gui.sfm.exit_without_saving_confirm.yes_button": "不保存退出",
+  "gui.sfm.facade_confirm_apply.message": "%d 种伪装状态将覆盖 %d 个方块。",
+  "gui.sfm.facade_confirm_apply.title": "确定要更新伪装外观吗？",
+  "gui.sfm.facade_confirm_change_world_block.message": "%d 个方块将改变形状，其伪装会被保留。",
+  "gui.sfm.facade_confirm_change_world_block.title": "确定要更改伪装的世界方块吗？",
+  "gui.sfm.facade_confirm_clear.message": "%d 种伪装状态将从 %d 个方块上清除。",
+  "gui.sfm.facade_confirm_clear.title": "确定要清除这些伪装吗？",
+  "gui.sfm.item_inspector.copied_to_clipboard": "已复制 {} 字符到剪贴板！",
+  "gui.sfm.label_gun.button.toggle_label_view": "循环标签视界",
+  "gui.sfm.label_gun.clear_button": "清除",
+  "gui.sfm.label_gun.label_button": "%s (%d)",
+  "gui.sfm.label_gun.label_edit_placeholder": "搜索或输入新标签",
+  "gui.sfm.label_gun.placeholder": "标签",
+  "gui.sfm.label_gun.prune_button": "清除冗余",
+  "gui.sfm.logs.button.clear_logs": "清空日志",
+  "gui.sfm.logs.button.clear_logs.packet_received": "日志已清空",
+  "gui.sfm.logs.button.copy_logs": "复制日志",
+  "gui.sfm.logs.button.copy_logs.tooltip": "Shift + 点击 以复制原始数据",
+  "gui.sfm.logs.empty_capability": "接收到空 Capability %s %s direction=%s",
+  "gui.sfm.logs.missing_adjacent_cable": "未找到 %s 附近的线缆",
+  "gui.sfm.logs.missing_capability_provider": "未找到 %s %s direction=%s 的 Capability Provider",
+  "gui.sfm.logs.no_content": "你好，世界！\n使用顶部按钮调整日志等级。\n高等级日志会在程序执行后重置。\n日志记录可能拖慢执行速度。\n滚动有问题，正在修复。\n目前仅显示数百行以减少卡顿。\n先用复制按钮在更好的编辑器中查看吧。",
+  "gui.sfm.logs.title": "日志",
+  "gui.sfm.manager.button.copy_to_clipboard": "复制到剪贴板",
+  "gui.sfm.manager.button.discord": "Discord",
+  "gui.sfm.manager.button.paste_clipboard": "从剪贴板粘贴",
+  "gui.sfm.manager.button.rebuild": "重建线缆网络",
+  "gui.sfm.manager.button.reset": "重置",
+  "gui.sfm.manager.button.server_config": "查看服务器配置",
+  "gui.sfm.manager.button.view_examples": "查看示例",
+  "gui.sfm.manager.button.view_examples.tooltip": "按 Ctrl + Shift + E 查看示例。",
+  "gui.sfm.manager.button.view_logs": "查看日志",
+  "gui.sfm.manager.button.warning.tooltip": "点击复制含警告和错误的代码。\nShift + 点击 以尝试修复警告。",
```

| **审查级别** |
|:-:|
| ℹ️ 信息 / 不阻止合并  |

**原文: Click to copy code with warnings and errors.
Shift-click to attempt to fix warnings.**

```suggestion
  "gui.sfm.manager.button.warning.tooltip": "点击复制含警告和错误的代码。\n按住 Shift 点击以尝试修复警告。",
```